voice/methods/conferences/getConference.md

```diff
@@ -0,0 +1,121 @@
+{% method %}
+
+## Retrieve Conference Information
+Retrieve the current state of a specific conference.
+
+### Request URL
+
+<code class="get">GET</code>`https://voice.bandwidth.com/api/v2/accounts/{accountId}/conferences/{conferenceId}`
```

Comment:
I would call this file `getConferencesConferenceId.md`, in case we have a `.../conferences'` endpoint in the future.